jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -362,6 +362,32 @@ private static PrintStream createConsole(FileDescriptor desc) {
 	/*[ENDIF] Sidecar18-SE-OpenJ9 */
 	/*[ENDIF] JAVA_SPEC_VERSION >= 11 */
 
+	static void initJFR() {
+		String jfr = com.ibm.oti.vm.VM.getjfrCMDLineOption();
+		if (jfr != null) {
+			try {
+				Class<?> dcmdClass = Class.forName("jdk.jfr.internal.dcmd.DCmdStart");
+				Constructor<?> constructor = dcmdClass.getDeclaredConstructor();
+				constructor.setAccessible(true);
+				Object dcmdInstance = constructor.newInstance();
+
+				Method executeMethod = dcmdClass.getSuperclass().getDeclaredMethod("execute", String.class, String.class, char.class);
+				executeMethod.setAccessible(true);
+
+				String[] results = (String []) executeMethod.invoke(dcmdInstance, "internal", jfr, ',');
+				if (results != null) {
+					for (String result : results) {
+						System.out.println(result);
+					}
+				}
+			} catch (ClassNotFoundException e) {
+				// Assume this is a raw configuration and suppress the exception
+			} catch (Exception e) {
+				throw new InternalError(e.toString());
```

Comment:
instead of just the string, you should pass the entire exception object `InternalError(e)` so there a stacktrace that indicates which operation failed